data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/ChangeEventStatus.java

```diff
@@ -21,6 +23,14 @@ public enum AcknowledgmentStatus {
 
     public ChangeEventStatus(final BinlogCoordinate binlogCoordinate, final long timestamp) {
         this.binlogCoordinate = binlogCoordinate;
+        this.logSequenceNumber = null;
```

Comment:
Why do we need this constructor ?

Reply:
In StreamCheckpointManager, we use a `ConcurrentLinkedQueue<ChangeEventStatus>` to track change event status. To make this `ChangeEventStatus` compatible for both MySQL and Postgres, I added logSequenceNumber in it, so MySQL will use binlogCoordinate and Postgres will use logSequenceNumber.